components/Activities/Activities.module.scss

```diff
@@ -35,7 +35,7 @@
     padding: 2em;
     display: flex;
     flex-direction: column;
-    justify-content: start;
+    justify-content: flex-start;
```

Comment:
What is the reason for this change?

Reply:
This warning: `start value has mixed support, consider using flex-start instead`